Source/WebCore/page/scrolling/ScrollingTreeFrameHostingNode.h

```diff
@@ -50,6 +55,7 @@ class ScrollingTreeFrameHostingNode : public ScrollingTreeNode {
     WEBCORE_EXPORT void dumpProperties(WTF::TextStream&, OptionSet<ScrollingStateTreeAsTextBehavior>) const override;
 
     std::optional<LayerHostingContextIdentifier> m_hostingContext;
+    HashSet<RefPtr<ScrollingTreeNode>> m_hostedChildren;
```

Comment:
Can you use a stronger type than `ScrollingTreeNode` ? Are they all frame scrolling nodes?

Reply:
Does this need to be able to hold null?  Can we use Ref instead of RefPtr?

Reply:
No these can be any scrolling tree node. Right this can be Ref.